internal/querywatcher/query_manager.go

```diff
@@ -51,8 +51,8 @@ type (
 		ResponseChan chan AdhocQueryResult
 	}
 
-	// QueryManager watches for changes in keys and notifies clients.
-	QueryManager struct {
+	// Manager watches for changes in keys and notifies clients.
+	Manager struct {
```

Comment:
I think it makes sense to have some context around manager, just like we have for worker and shards.

Reply:
I feel that since everywhere this struct is referenced outside of this package - it will carry the package qualifier - `querymanager.Manager`, it would be clear enough.
`querymanager.QueryManager` feels a little repetitive.

No strong opinion around this though.